Tools/OSD/osd_screen.cs

```diff
@@ -604,12 +606,6 @@ private void rbtSortAlphabetic_CheckedChanged(object sender, EventArgs e)
 			}
 		}
 		
-        private int correct_NTSC(int y){
-            if (y >= osd.getCenter() && osd.is_ntsc()) 
-                y+=3;
-            return y;
-        }
-
```

Comment:
Ok, then do it in your way (Or not). And keep being sarcastic that PRs are welcome